ghttp/ghttpProcess.c

```diff
@@ -1208,7 +1208,7 @@ static GHTTPBool ghiDeliverIncomingFileData
 	{
 		int bytesWritten = 0;
 #ifndef NOFILE
-		bytesWritten = fwrite(data, 1, len, connection->saveFile);
+		bytesWritten = (int)fwrite(data, 1, len, connection->saveFile);
```

Comment:
doesn't fwrite return size_t?

Reply:
`len` is `int`, `count` is `1`, so `1 * len` is always `int`. 
`bytesWritten` <= `1 * len` so decided to be less intrusive and use `int`.

May change to `size_t` for `len` argument and so on, but that requires more changes.

Reply:
hmm by giving a second look, the people at gamespy decided to use all the lengths as "int", so it doesn't currently make a lot of sense to use size_t atm, it's ok to keep it this way